### DISCUSSION TITLE
Question---

### DISCUSSION THREAD

**Discussion by mustafadem1ir** [SoftwareBrothers/admin-bro#1517] [Q&A]
How do we determine the file path for import-export in a component loader?

**Comment by ylmzfurkan** [comment]
Relative Paths: In most component loaders, you can import or export files using relative paths based on the current file or module location. A relative path is specified relative to the directory structure of the file.
For example, if you have the following directory structure:

```
- components/
  - myComponent.js
- utils/
  - helper.js

```

To import the helper.js file from the myComponent.js file, you can use a relative path like this:

`import { someFunction } from '../utils/helper.js';`

**Comment by mustafadem1ir** [comment]
Thanksss
